edge-modules/azure-monitor/src/IothubConnectionManager.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Client;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.Devices.Edge.Azure.Monitor
+{
+
+    public class IothubConnectionManager
+    {
+        ModuleClientWrapper ModuleClientWrapper;
+        SemaphoreSlim Semaphore;
+
+        public IothubConnectionManager(ModuleClientWrapper moduleClient, SemaphoreSlim semaphore)
+        {
+            this.ModuleClientWrapper = moduleClient;
+            this.Semaphore = semaphore;
+        }
+
+        public async Task ConnectToIothub(CancellationToken ct)
+        {
+            await this.Semaphore.WaitAsync();
+            Console.WriteLine($"connection manager before recreate {this.ModuleClientWrapper.Inner}");
```

Comment:
Are these before/after log statements printing out something useful? I'm guessing the output looks something like:

```
connection manager before recreate Microsoft.Azure.Devices.Client.ModuleClient
connection manager after recreate Microsoft.Azure.Devices.Client.ModuleClient
```

If that's the case we should remove.

Reply:
Good catch, thought I removed all the debug logging but I hadn't. Doing a once over to confirm it is all gone.